trajopt_ifopt/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>trajopt_ifopt</name>
+  <version>0.1.0</version>
+  <description>The trajopt_ifopt package</description>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong</maintainer>
+  <license>Apache 2.0</license>
+  <author>Levi Armstrong</author>
+  <author>Matthew Powelson</author>
+
+  <depend>ifopt</depend>
+  <depend>libconsole-bridge-dev</depend>
+  <depend>trajopt</depend>
```

Comment:
I think it is missing a few depends based on the cmake find_package.

find_package(trajopt_utils)
find_package(trajopt)
find_package(console_bridge REQUIRED)
find_package(Eigen3 REQUIRED)
find_package(ifopt REQUIRED)
find_package(tesseract REQUIRED)